<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Help</title>
    <link href="css/common.css" rel="stylesheet">
    <script type="text/javascript" src="js/jquery-3.7.1.min.js"></script>
</head>
<body>
<div class="body_wrap">
    <nav class="nav_bar">
        <div class="nav_bar_cont">
            <div class="logo">
                <a href="main.html">BreastASMiner</a>
            </div>
            <ul class="nav">
                <li><a href="Single-cancer_AS.html">Data Exploring</a></li>
                <li><a href="">Subtype-specific AS</a></li>
                <li><a href="">Gene Search</a></li>
                <li><a href="download.html">Download</a></li>
                <li><a href="help.html">Help</a></li>
                <li><a href="contact.html">Contacts</a></li>
            </ul>
        </div>
    </nav>

    <section class="container_wrap help_wrap">
        <div class="content_title_wrap">
            <div class="content_title">
                <h1>Help</h1>
                We provide user manual to help users use our data easily
            </div>
        </div>

        <div class="help_cont_wrap">
            <div class="help_nav">
                <ul>
                    <li><a href="#help1" class="active">1. Introduction</a></li>
                    <li><a href="#help2">2. Alternative Splicing Events</a></li>
                    <li><a href="#help3">3. TCGA molecular subtype information</a></li>
                    <li><a href="#help4">4. Significant Subtype-Specific AS Events</a></li>
                    <li><a href="#help5">5. Phenotype association</a></li>
                    <li>
                        <a href="#help6">6. Single-cancer AS</a>
                        <ul class="sub">
                            <li><a href="#help6-1">6-1. AS-level</a></li>
                            <li><a href="#help6-2">6-2. Patient-level</a></li>
                            <li><a href="#help6-3">6-3. Tissue-level</a></li>
                            <li><a href="#help6-4">6-4. Gene-level</a></li>
                        </ul>
                    </li>
                    <li>
                        <a href="#help7">7. Pan-cancer AS</a>
                        <ul class="sub">
                            <li><a href="#help7-1">7-1. AS-level</a></li>
                            <li><a href="#help7-2">7-2. Patient-level</a></li>
                            <li><a href="#help7-3">7-3. Gene-level</a></li>
                        </ul>
                    </li>
                    <li>
                        <a href="#help8">8. Others</a>
                        <ul class="sub">
                            <li><a href="#help8-1">8-1. Genes related biological pathways</a></li>
                            <li><a href="#help8-2">8-2. Case study</a></li>
                        </ul>
                    </li>
                </ul>
            </div>
            <div class="help_cont">
                <div id="help1">
                    <h2>1. Introduction</h2>
                    <p><strong class="point01">AS-CMC</strong> (Alternative Splicing for Cancer Molecular Classification) is a web-based database for allowing users to browse subtype-specific changes in AS along with phenotypic association for each cancer type as well as compare the regulation pattern across diverse cancer types. For AS in TCGA samples, we used the per-cent-spliced-in index (PSI) value from TCGASpliceSeq database. We obtained the information pertaining to cancer molecular subtypes from TCGAbiolinks R package.</p>
                </div>

                <div id="help2">
                    <h2>2. Alternative Splicing Events</h2>
                    <h3>Alternative Splicing Events</h3>
                    <p>For AS of TCGA samples, we download the PSI value of <strong>27,682</strong> AS events in <strong>24</strong> cancer types from <strong>the TCGASpliceSeq database</strong>. AS-CMC provided the PSI values of five splicing types: exon skip (ES), retained intron (RI), alternate acceptor sites (AA), alternate donor sites (AD), and mutually exclusive exon (ME).</p>
                    <div align="center">
                        <img src="images/img_help2_1.svg" alt="" width="670">
                    </div>
                    <h3>Quantification of AS Events</h3>
                    <p><strong class="point01">Percent splice in (PSI)</strong> ranged from 0 to 1 was a commonly used ratio to indicate different uses of alternative exon. For each splice event, a percent-splice-in (PSI) value generated by the ratio of inclusion of reads over the total reads for that event (both inclusion and exclusion reads).</p>
                    <div align="center">
                        <img src="images/img_help2_2.svg" alt="" width="670">
                    </div>
                    <h3>AS ID Definition</h3>
                    <p>AS ID rule is simple to create from gene model. AS ID format includes a AS type, exon number and delimiting character '_'.
                        <br><strong>(AS ID : Gene.Name_AS.Type_Retulated.Exon_From.Exon_To.Exon)</strong></p>
                    <div align="center">
                        <img src="images/img_help2_3.svg" alt="" width="670">
                    </div>
                    <p class="refer"><strong>Reference</strong> SpliceSeq: a resource for analysis and visualization of RNA-Seq data on alternative splicing and its functional impacts. Bioinformatics. 2012;28(18):2385-7</p>
                </div>
                
                <div id="help3">
                    <h2>3. TCGA molecular subtype information</h2>
                    <p>The information of cancer molecular sub-types was obtained from <a href="#">TCGAbiolinks R package</a>. And we used the most prominent subtype classification which is curated annotation named ‘<strong>Subtype_Selected</strong>’. For more information on the molecular types of each cancer, see the literature links in the table below.</p>
                    
                    <table style="border-collapse:separate;border-spacing:3px; border:2px solid grey;font-family:Arial" class="table_3735" border="5">
                        <colgroup><col width="60">
                        <col width="150">
                        <col width="120">
                        <col width="60">
                        <col width="1100">
                        </colgroup><thead>
                        <tr style="background-color:white;">
                          <th id="tableHTML_header_1"> Cancer</th>
                          <th id="tableHTML_header_2"> Molecular data</th>
                          <th id="tableHTML_header_3"> Reference</th>
                          <th id="tableHTML_header_4"> #Type</th>
                          <th id="tableHTML_header_5"> Molecular Subtype (n)</th>
                        </tr>
                        </thead>
                        <tbody>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">ACC</td>
                          <td id="tableHTML_column_2">DNAmeth</td>
                          <td id="tableHTML_column_3"><a href="http://www.cell.com/cancer-cell/fulltext/S1535-6108(16)30160-X" target="_blank">Cancer Cell 2016 </a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">3</td>
                          <td id="tableHTML_column_5">ACC.CIMP-high, ACC.CIMP-intermediate, ACC.CIMP-low</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">BLCA</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="http://www.nature.com/nature/journal/v507/n7492/full/nature12965.html" target="_blank">Nature 2014</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">BLCA.4, BLCA.3, BLCA.1, BLCA.2</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">BRCA</td>
                          <td id="tableHTML_column_2">PAM50(mRNA)</td>
                          <td id="tableHTML_column_3"><a href="http://www.nature.com/nature/journal/v490/n7418/full/nature11412.html#close" target="_blank">Nature 2012</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">5</td>
                          <td id="tableHTML_column_5">BRCA.Normal, BRCA.Her2, BRCA.Basal, BRCA.LumB, BRCA.LumA</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">COAD</td>
                          <td id="tableHTML_column_2">Molecular_Subtype</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(18)30114-4" target="_blank">Cancer Cell 2018</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">GI.HM-SNV, GI.GS, GI.MSI, GI.CIN</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">ESCA</td>
                          <td id="tableHTML_column_2">Molecular_Subtype</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(18)30114-4" target="_blank">Cancer Cell 2018</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">5</td>
                          <td id="tableHTML_column_5">GI.GS, GI.HM-SNV, GI.MSI, GI.CIN, GI.ESCC</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">GBM</td>
                          <td id="tableHTML_column_2">Supervised_DNAmeth</td>
                          <td id="tableHTML_column_3"><a href="http://www.cell.com/cell/abstract/S0092-8674(15)01692-X" target="_blank">Cell 2016</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">6</td>
                          <td id="tableHTML_column_5">GBM_LGG.G-CIMP-high, GBM_LGG.G-CIMP-low, GBM_LGG.LGm6-GBM, GBM_LGG.NA, GBM_LGG.Classic-like, GBM_LGG.Mesenchymal-like</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">HNSC</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="https://www.nature.com/articles/nature14129" target="_blank">Nature 2015</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">HNSC.Classical, HNSC.Atypical, HNSC.Mesenchymal, HNSC.Basal</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">KICH</td>
                          <td id="tableHTML_column_2">Eosinophilic</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(14)00304-3" target="_blank">Cancer Cell 2014</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">2</td>
                          <td id="tableHTML_column_5">KICH.Eosin.1, KICH.Eosin.0</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">KIRC</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(14)00304-3" target="_blank">Nature 2013</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">5</td>
                          <td id="tableHTML_column_5">KIRC.NA, KIRC.4, KIRC.2, KIRC.3, KIRC.1</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">KIRP</td>
                          <td id="tableHTML_column_2">COC</td>
                          <td id="tableHTML_column_3"><a href="http://www.nejm.org/doi/full/10.1056/NEJMoa1505917#t=article" target="_blank">NEJM 2015</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">KIRP.C2c - CIMP, KIRP.C2b, KIRP.C2a, KIRP.C1</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">LAML</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="https://www.nejm.org/doi/full/10.1056/NEJMoa1301689" target="_blank">NEJM 2013</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">7</td>
                          <td id="tableHTML_column_5">AML.1, AML.3, AML.7, AML.2, AML.5, AML.6, AML.4</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">LGG</td>
                          <td id="tableHTML_column_2">Supervised_DNAmeth</td>
                          <td id="tableHTML_column_3"><a href="http://www.cell.com/cell/abstract/S0092-8674(15)01692-X" target="_blank">Cell 2016</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">7</td>
                          <td id="tableHTML_column_5">GBM_LGG.NA, GBM_LGG.G-CIMP-low, GBM_LGG.Classic-like, GBM_LGG.PA-like, GBM_LGG.Mesenchymal-like, GBM_LGG.Codel, GBM_LGG.G-CIMP-high</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">LIHC</td>
                          <td id="tableHTML_column_2">iCluster</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cell/fulltext/S0092-8674(17)30639-6" target="_blank">Cell 2017</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">LIHC.NA, LIHC.iCluster:2, LIHC.iCluster:3, LIHC.iCluster:1</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">LUAD</td>
                          <td id="tableHTML_column_2">iCluster</td>
                          <td id="tableHTML_column_3"><a href="http://www.nature.com/nature/journal/v511/n7511/full/nature13385.html" target="_blank">Nature 2014</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">6</td>
                          <td id="tableHTML_column_5">LUAD.1, LUAD.2, LUAD.4, LUAD.6, LUAD.3, LUAD.5</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">LUSC</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="http://www.nature.com/nature/journal/v489/n7417/full/nature11404.html" target="_blank">Nature 2012</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">LUSC.primitive, LUSC.basal, LUSC.secretory, LUSC.classical</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">OV</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="https://www.nature.com/articles/nature10166" target="_blank">Nature 2011</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">OVCA.Immunoreactive, OVCA.Mesenchymal, OVCA.Differentiated, OVCA.Proliferative</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">PCPG</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="http://www.cell.com/cancer-cell/fulltext/S1535-6108(17)30001-6" target="_blank">Cancer Cell 2017</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">5</td>
                          <td id="tableHTML_column_5">PCPG.NA, PCPG.Cortical admixture, PCPG.Wnt-altered, PCPG.Pseudohypoxia, PCPG.Kinase signaling</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">PRAD</td>
                          <td id="tableHTML_column_2">Mutation/Fusion</td>
                          <td id="tableHTML_column_3"><a href="http://www.cell.com/cell/abstract/S0092-8674(15)01339-2" target="_blank">Cell 2015</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">8</td>
                          <td id="tableHTML_column_5">PRAD.7-IDH1, PRAD.4-FLI1, PRAD.6-FOXA1, PRAD.3-ETV4, PRAD.2-ETV1, PRAD.5-SPOP, PRAD.8-other, PRAD.1-ERG</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">READ</td>
                          <td id="tableHTML_column_2">Molecular_Subtype</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(18)30114-4" target="_blank">Cancer Cell 2018</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">4</td>
                          <td id="tableHTML_column_5">GI.MSI, GI.HM-SNV, GI.GS, GI.CIN</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">SKCM</td>
                          <td id="tableHTML_column_2">Mutation</td>
                          <td id="tableHTML_column_3"><a href="http://www.cell.com/cell/abstract/S0092-8674(15)00634-0" target="_blank">Cell 2015</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">5</td>
                          <td id="tableHTML_column_5">SKCM.NF1_Any_Mutants, SKCM.Triple_WT, SKCM.- , SKCM.RAS_Hotspot_Mutants, SKCM.BRAF_Hotspot_Mutants</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">STAD</td>
                          <td id="tableHTML_column_2">Molecular_Subtype</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(18)30114-4" target="_blank">Cancer Cell 2018</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">5</td>
                          <td id="tableHTML_column_5">GI.HM-SNV, GI.EBV, GI.GS, GI.MSI, GI.CIN</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">THCA</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)01238-0#" target="_blank">Cell 2014</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">6</td>
                          <td id="tableHTML_column_5">THCA.NA, THCA.2, THCA.3, THCA.5, THCA.4, THCA.1</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">UCEC</td>
                          <td id="tableHTML_column_2">iCluster - updated according to Pan-Gyne/Pathways groups</td>
                          <td id="tableHTML_column_3"><a href="http://www.nature.com/nature/journal/v497/n7447/full/nature12113.html" target="_blank">Nature 2013</a></td>
                          <td id="tableHTML_column_4" style="text-align:center;">5</td>
                          <td id="tableHTML_column_5">UCEC.NA, UCEC.POLE, UCEC.MSI, UCEC.CN_LOW, UCEC.CN_HIGH</td>
                        </tr>
                        <tr>
                          <td id="tableHTML_column_1">UCS</td>
                          <td id="tableHTML_column_2">mRNA</td>
                          <td id="tableHTML_column_3"><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(17)30053-3" target="_blank">Cancer Cell 2017</a></td><td id="tableHTML_column_4" style="text-align:center;">2</td>
                          <td id="tableHTML_column_5">UCS.1, UCS.2</td>
                        </tr>
                        </tbody>
                    </table>
                    
                    <p class="refer"><strong>Reference</strong> TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data. Nucleic Acids Res. 2016 May 5;44(8):e71</p>
                </div>
                
                <div id="help4">
                    <h2>4. Significant Molecular Subtype-Specific AS Events</h2>
                    <h3>ANOVA test</h3>
                    <p>We performed ANOVA test to selected subtype-specific AS events by analysis of variance <strong>(p-value < 1x10 -3)</strong> for each cancer type. For visualization,AS-CMC displays boxplots for a each cancer and a scatter plot across cancers.</p>
                    <div align="center">
                        <img src="images/img_help4_1.svg" alt="" width="490">
                    </div>
                    <h3>Each cancer</h3>
                    <div align="center">
                        <img src="images/img_help4_2.svg" alt="" width="365">
                    </div>
                    <h3>Pan cancers</h3>
                    <div align="center">
                        <img src="images/img_help4_3.svg" alt="" width="750">
                    </div>
                </div>
                
                <div id="help5">
                    <h2>5. Phenotype association (Tissue-level & Gene-level)</h2>
                    <p>The “Phenotype association” module enables prioritization of AS events by relevance in terms of clinical outcomes (patient-level), cancer microenvironment scores (tissue-level), and gene expression levels (gene-level).

                    In this section show two workflows(Tissue-level & Gene-level) of a module. <strong>Tissue-level</strong>: In the cancer microenvironment part,users can investigate the correlations between the changes in AS and predefined scores indicating the status of immune and stromal cells, epithelial-to-mesenchymal transition (EMT), and hypoxia. <strong>Gene-level</strong>: Users can also examine the correlations of each AS event with expression levels of all genes. The gene expression levels correlated with AS PSI values enables determination of biological pathways underlying AS changes. The relationship of the host gene with AS helps in assessing the dependency of the AS event on transcriptional regulation. If the AS is independent of gene expression, it is likely regulated solely by splicing machinery</p>
                    <div align="center">
                        <img src="images/img_help5.png" alt="" width="900">
                    </div>
                </div>
                
                <div id="help6">
                    <h2>6. Single-cancer AS</h2>
                    <p>This page shows the approches for the cancer. Method for searching a significant molecular subtype-specific AS events in certain cancer is as follows.</p>
                    <div align="center">
                        <img src="images/img_help6_1.svg" alt="" width="800">
                    </div>
                    <p><strong>a.</strong> List of subtype-specific AS events for a selected cancer type. Once a cancer type is selected, the subtype-specific AS events are shown with relevant statistics displayed in a tabular form. Users can filter the results using the defined cut-off in survival and correlation with the expression level of genes with the AS. <strong>b.</strong> Visualization panel showing the analysis results of an AS event. Once an AS event is selected (click the hyper linked letter named "Click"), a window with various plots pops up.</p>
                </div>
                
                <div id="help6-1">
                    <h2>6-1. AS-level (Significant AS events)</h2>
                    <p>We performed survival analysis between high PSI and low PSI cohorts. Threshold PSI value to separate two groups was defined as 10%, 25%, and 50% of PSI value in the distribution. The significance for differential survival rates was evaluated by Log-rank test. Three thresholds in PSI value distribution were applied to generate two groups to be tested. P-value derived from Log-rank test. A pan-cancer view of survival analysis results for the queried AS events.</p>
                    <div align="center">
                        <img src="images/img_help6_2.svg" alt="" width="300">
                    </div>
                    <p>We compared exon-level PSI values and gene expression levels across cancer molecular subtypes. And we checked the correlation of PSI value to spliced gene expression across cancer molecular subtypes.</p>
                    <div align="center">
                        <img src="images/img_help6_3.svg" alt="" width="900">
                    </div>
                    <p class="refer"><strong>Reference</strong> TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data. Nucleic Acids Res. 2016 May 5;44(8):e71</p>
                </div>
                
                <div id="help6-2">
                    <h2>6-2. Patient-level (Association of survival with AS events)</h2>
                    <p>To identify potential clinically association of a significant AS event, we analysis survival analysis using TCGA survival data. .</p>
                    <h3>Survival Analysis</h3>
                    <p>We performed survival analysis between high PSI and low PSI cohorts. Threshold PSI value to separate two groups was defined as 10%, 25%, and 50% of PSI value in the distribution. The significance for differential survival rates was evaluated by Log-rank test. Three thresholds in PSI value distribution were applied to generate two groups to be tested. P-value derived from Log-rank test. A pan-cancer view of survival analysis results for the queried AS events.</p>
                    <div align="center">
                        <img src="images/img_help6_4.svg" alt="" width="900">
                    </div>
                    <p class="refer"><strong>Reference</strong> TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data. Nucleic Acids Res. 2016 May 5;44(8):e71</p>
                </div>
                
                <div id="help6-3">
                    <h2>6-3. Tissue-level (Association of tissue-level score)</h2>
                    <p>To test the relationship of each AS event with tumor microenvironment, we used immune infiltration levels, tumor purity,hypoxia and EMT, which were previously published in pan-cancer analysis. First, we checked if PSI values are related to tumor microenvironment using cellular fraction estimates (leukocyte fraction and CIBERSORT immune fractions) [1]. Next, we assesed the association of PSI values with tumor hypoxia scores which can reflect the level of molecular oxygen in tumor sample [2]. Finally, we provided the association of EMT value [3].</p>
                    <div align="center">
                        <img src="images/img_help6_5.svg" alt="" width="900">
                    </div>
                    <p class="refer">
                        <strong>Reference</strong><br>
                        1) The spliceosome pathway activity correlates with reduced anti-tumor immunity and immunotherapy response, and unfavorable clinical outcomes in pan-cancer. Comput Struct Biotechnol J 2021;19:5428-5442<br>
                        2) The Immune Landscape of Cancer. Immunity 2018;48:812-30 e14<br>
                        3) Molecular landmarks of tumor hypoxia across cancer types. Nat Genet 2019;51:308-18<br>
                        4) Pan-cancer survey of epithelial-mesenchymal transition markers across the Cancer Genome Atlas. Dev Dyn 2018;247:555-64
                    </p>
                </div>

                <div id="help6-4">
                    <h2>6-4. Gene-level (Association of Gene Expression with AS Events)</h2>
                    <p>We explored which genes have mRNA levels correlated with queried AS.</p>
                    <div align="center">
                        <img src="images/img_help6_6.svg" alt="" width="900">
                    </div>
                </div>

                <div id="help7">
                    <h2>7. Pan-cancer AS</h2>
                    <p>This page shows the approches for the cancer.Method for searching a significant molecular subtype-specific AS events in certain cancer is as follows.</p>
                    <div align="center">
                        <img src="images/img_help7_1.svg" alt="" width="800">
                    </div>
                    <p><strong>a.</strong> Selection panel for AS events. Users are directed to the pan-cancer views upon clicking (click the hyper linked letter named "Click") on an AS event. <strong>b-d.</strong> Pan-cancer information pertaining to MAP3K7 exon 11 as an example. <strong>b.</strong> Comparison of subtype-specificity across cancer types. ANOVA results are displayed using two different y-axes: -log10.</p>
                </div>

                <div id="help7-1">
                    <h2>7-1. AS-level (Significant AS events)</h2>
                    <p>We performed ANOVA test to selected subtype-specific AS events by analysis of for each cancer type. And then we identify the distribuion p & r-value across cancers</p>
                    <div align="center">
                        <img src="images/img_help7_2.svg" alt="" width="900">
                    </div>
                    <p class="refer"><strong>Reference</strong> TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data. Nucleic Acids Res. 2016 May 5;44(8):e71</p>
                </div>

                <div id="help7-2">
                    <h2>7-2. Patient-level (Association of Survival with AS Events)</h2>
                    <h3>Survival Analysis</h3>
                    <p>We performed survival analysis between high PSI and low PSI cohorts. Threshold PSI value to separate two groups was defined as 10%, 25%, and 50% of PSI value in the distribution. The significance for differential survival rates was evaluated by Log-rank test. Three thresholds in PSI value distribution were applied to generate two groups to be tested. P-value derived from Log-rank test. A pan-cancer view of survival analysis results for the queried AS events.</p>
                    <div align="center">
                        <img src="images/img_help7_3.svg" alt="" width="600">
                    </div>
                    <p class="refer"><strong>Reference</strong> TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data. Nucleic Acids Res. 2016 May 5;44(8):e71</p>
                </div>

                <div id="help7-3">
                    <h2>7-3. Gene-level (Association of gene expression with AS events)</h2>
                    <div align="center">
                        <img src="images/img_help7_4.png" alt="" width="800">
                    </div>
                </div>

                <div id="help8-1">
                    <h2>8-1. Genes related biological pathways</h2>
                    <p>Display genes of significant ASE events across cancers</p>
                    <table style="border-collapse:separate;border-spacing:2px;border:1px solid grey;" class="table_4074" border="1" align="center">
                        <thead>
                        <tr style="background-color:white;">
                          <th id="tableHTML_header_1">Pathways</th>
                          <th id="tableHTML_header_2">Genes</th>
                        </tr>
                        </thead>
                        <tbody>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">Cell Cycle Control (3 genes)</td>
                          <td id="tableHTML_column_2">CDK2, E2F5, E2F6</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">Notch signaling (11 genes)</td>
                          <td id="tableHTML_column_2">APH1A,  ARRDC1, DLL3, DTX2, DTX3, HES4, ITCH, JAG2, NCOR2, NUMB, RBPJ</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">DNA Damage Response (4 genes)</td>
                          <td id="tableHTML_column_2">CHEK1, CHEK2, RAD51, MLH1</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">Other growth/proliferation signaling (4 genes)</td>
                          <td id="tableHTML_column_2">FGFR1, CSF1, PLAGL1, AURKA</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">Survival/cell death regulation signaling (4 genes)</td>
                          <td id="tableHTML_column_2">BCL2L1, BCL2, CASP10, CASP3</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">RTK signaling family (3 genes)</td>
                          <td id="tableHTML_column_2">FGFR1, VEGFA, PDGFA</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">PI3K-AKT-mTOR signaling (4 genes)</td>
                          <td id="tableHTML_column_2">TSC2, MLST8, AKT1, RHEB</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">Ras-Raf-MEK-Erk/JNK signaling (8 genes)</td>
                          <td id="tableHTML_column_2">MAPK9, HRAS, MAP3K4, KRAS, MAPK3, MAP3K3, DAB2, MAPK8</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">Regulation of ribosomal protein synthesis and cell growth (3 genes)</td>
                          <td id="tableHTML_column_2">RPS6, RPS6KB2, RPS6KB1</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">Angiogenesis (1 gene)</td>
                          <td id="tableHTML_column_2">VEGFA</td>
                        </tr>
                        <tr style="background-color:#e6f0ff;">
                          <td id="tableHTML_column_1">Invasion nad metastatsis (4 genes)</td>
                          <td id="tableHTML_column_2">MMP23B, PTK2, MMP19, WFDC2</td>
                        </tr>
                        <tr style="background-color:white;">
                          <td id="tableHTML_column_1">TGF-beta Pathway (2 genes)</td>
                          <td id="tableHTML_column_2">TGFBR3, SMAD5</td>
                        </tr>
                        </tbody>
                    </table>
                    <p class="refer"><strong>Reference</strong> https://www.cbioportal.org/</p>
                </div>

                <div id="help8-2">
                    <h2>8-2. Case study: MAP3K7_ES_11</h2>
                    <p>Using AS-CMC, we selected a notable subtype-specific AS event, which can serve as a pan-cancer AS biomarker as an example. An ES event in exon 11 of MAP3K7 (mitogen-activated protein kinase kinase kinase 7) gene was chosen as this marker showed significant subtype-specificity in 10 cancer types (BRCA, LGG, ESCA, STAD, HNSC, BLCA, OV, THCA, LUAD, UCS). In the survival analysis, the skipping of the exon was found to be strongly associated with poor clinical outcome in stomach adenocarcinoma (STAD). The survival difference was found to be the largest in the most stringent cut-off (upper 10% vs lower 10%).

                    MAP3K7 AS was analyzed in depth in STAD due to its significant association with survival. Among the five molecular subtypes of STAD, only GI.GS subtype showed significant distribution of PSI values compared to the other subtypes. The PSI values were correlated with the molecular scores related to EMT scores (r = -0.71). Taken together, these data support that an ES event in exon 11 of MAP3K7 may play a role in regulating subtypes across diverse cancers, and that this event may particularly play a crucial role in STAD where its function has not been reported earlier.</p>
                    <div align="center">
                        <img src="images/img_help8.png" alt="" width="900">
                    </div>
                    <p>An example of a potential pan-cancer AS biomarker. (a) Location of MAP3K7 exon with AS on the corresponding gene and chromosome. (b) Distribution of PSI values for five molecular subtypes in STAD. (c) Correlations between the AS PSIs and EMT scores in STAD. Spearman’s correlation coefficient is shown on the top. Each dot represents an individual patient sample. (d) Survival plots comparing survival rates between two patient groups with high- and low-PSI values in STAD. In the survival plots, AS-CMC provides three plots representing the survival difference between the groups for an AS event according to the following PSI cut-offs: 50% (upper 50% vs lower 50%), 25% (upper 25% vs lower 25%), and 10% (upper 10% vs lower 10%). The significance of differential survival rates was evaluated using log rank test.</p>
                </div>
            </div>
        </div>
    </section>
</div>

<script>
var sections = $('.help_cont > div')
  , nav = $('.help_nav')
  , nav_height = nav.outerHeight();

$(window).on('scroll', function () {
  var cur_pos = $(this).scrollTop();
  
  sections.each(function() {
    var top = $(this).offset().top - 150,
        bottom = top + 140;
    
    if (cur_pos >= top && cur_pos <= bottom) {
      nav.find('a').removeClass('active');
      sections.removeClass('active');
      
      $(this).addClass('active');
      nav.find('a[href="#'+$(this).attr('id')+'"]').addClass('active');
    }
  });
});

nav.find('a').on('click', function () {
  var $el = $(this)
    , id = $el.attr('href');
  
  $('html, body').animate({
    scrollTop: $(id).offset().top - 140
  }, 500);
  
  return false;
});
</script>
</body>
</html>